bin/post_compile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```

Comment:
This script only makes sense for heroku, correct? If so, does it have to be called `post_compile` or can we make it clear it is  for Heroku only ?